src/live_restore/live_restore_state.c

```diff
@@ -369,21 +369,37 @@ __wti_live_restore_validate_directories(WT_SESSION_IMPL *session, WTI_LIVE_RESTO
     char **dirlist_source = NULL, **dirlist_dest = NULL;
     uint32_t num_source_files = 0, num_dest_files = 0;
     WTI_LIVE_RESTORE_STATE state_from_file;
+    bool contain_backup_file = false;
 
-    /* First check that the source doesn't contain any live restore metadata files. */
+    /*
+     * First check that the source doesn't contain any live restore metadata files, but does contain
```

Comment:
Remind me what a live restore metadata file is?

Reply:
This was written back when the `WiredTigerLR.state` file and per-file bitmaps might have been stored in the destination. These days it just means stop files so I've fixed the wording in [21a413a](https://github.com/wiredtiger/wiredtiger/pull/11757/commits/21a413af3aee92152e793d44bbaadaf618911a35)

Reply:
Great, thanks